.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,3 +38,16 @@ The changelogs will be integrated by the core maintainers after the merge.  See
 <!-- Comment:
 If you need an accelerated review process by the community (e.g., for critical bugs), mention @jenkinsci/code-reviewers
 -->
+
+### Maintainer checklist
+
+Before the changes is marked as `ready-for-merge`: 
+
+- [ ] There is at least 2 approvals for the pull request and no outstanding requests for change
+- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
+- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
+- [ ] Proper changelog labels are set so that the changelog can be generated automatically
+- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
```

Comment:
Isn't it better to add these sections to the template and mark it as _optional_? Because it's under the _Maintainer checklist_ and maybe the creator doesn't look at it

Reply:
Yes, I will add it explicitly to the template

Reply:
Done